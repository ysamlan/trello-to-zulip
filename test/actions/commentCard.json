{
  "data": {
    "board": {
      "id": "b1idb1idb1idb1idb1idb1id",
      "name": "Board One"
    },
    "card": {
      "id": "cidcidcidcidcidcidcidcid",
      "idShort": 1,
      "name": "Card Name",
      "shortLink": "cslcslss"
    },
    "text": "Some data text that has a number of words in it."
  },
  "date": "2013-11-03T20:23:46.185Z",
  "id": "actidactidactidactidacti",
  "idMemberCreator": "mcidmcidmcidmcidmcidmcid",
  "memberCreator": {
    "avatarHash": "mcahmcahmcahmcahmcahmcahmcahmcah",
    "fullName": "Member Creator Full Name",
    "id": "mcidmcidmcidmcidmcidmcid",
    "initials": "MCFN",
    "username": "mcfname"
  },
  "type": "commentCard"
}