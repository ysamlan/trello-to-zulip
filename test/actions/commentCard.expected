Member Creator Full Name commented on card [Card Name](https://trello.com/c/cidcidcidcidcidcidcidcid) 
>Some data text that has a number of words in it.